media/js/base/mozilla-global-nav.js

```diff
@@ -61,6 +61,12 @@
          * @param {String} id - optional accordion id to expand on open.
          */
         toggleDrawer: function(accordionId) {
+
+            // make sure we're at the top of the window before opening drawer.
```

Comment:
This is a workaround to fix a bug in the global nav when there is an alement that comes above it in the page